schemas/org.gnome.shell.extensions.paperwm.gschema.xml

```diff
@@ -393,6 +393,11 @@
         <summary>Show the focus mode icon in the topbar</summary>
     </key>
 
+    <key type="b" name="allow-all-actions-in-navigator">
```

Comment:
Do we need this to be an user settable option (with a default of false)?

Is there a reason why we wouldn't want this behaviour by default?  e.g. I can't think of a reason why would not want other shortcuts executable while minimap is open.

Reply:
I don't know the original reason for separating the keybindings but IMO we can also just change the behavior and not create a user setting for this.

The only breakage I can think of (not tested) is that I think it would be possible (in user.js) to create a keybinding that is only active in the navigator but has the same keys as another keybinding. But I think you can work around that because the handler get passed `navigator` and I think you also globally check if the navigator is open in your custom handler.

Reply:
Agree, would make this default behaviour.  Test out your hypothesis (as much as you can) - but with `user.js`, users can implement unintended things/behaviours (including code that would completely break PaperWM) so  I wouldn't let concerns about users potentially implementing something in there stop us from simplifying, testing, and improving this.

Make changes you see fit (e.g. making this default) and I can help testing widely and then merge.

Reply:
I removed the option and just made this the new behavior.